assets/css/_info-bubble.scss

```diff
@@ -181,18 +181,16 @@
 
   .non-variant {
     float: right;
-    margin-right: 8px;
-    margin-top: 5px;
-    margin-bottom: 8px;
-    width: 150px;
-    height: $info-bubble-button-size;
-    line-height: $info-bubble-button-size;
-
-    user-select: none;
-
-    white-space: nowrap;
-
     text-align: right;
+    margin-top: 9px;
```

Comment:
OK, I have it working for Chrome and Firefox, but I cannot figure out a way to target Safari specifically since it is showing up differently than the rest.